src/actions/Actions.js

```diff
@@ -0,0 +1,25 @@
+import { createActions } from 'redux-actions'
+
```

Comment:
These new lines that can be removed.
Try and keep it consistent with a single breakline when needed.